activestorage/lib/active_storage/previewer.rb

```diff
@@ -27,7 +27,7 @@ def preview(**options)
     end
 
     private
-      # Downloads the blob to a tempfile on disk. Yields the tempfile.
+      # Downloads the blob to a tempfile on disk. See +ActiveStorage::Blob#open+ for details.
```

Comment:
```suggestion
      # Downloads the blob to a tempfile on disk. See ActiveStorage::Blob#open for details.
```

If you want it to auto-link need to remove the fixed-width wrapper.

Reply:
Done!